css/BEM-style.md

```diff
@@ -0,0 +1,223 @@
+If that's not worse, how about you also finding out that your new styles written lack behind on specificity and your styles were never applied, because some other previously applied class or even id made your previous styles highly specific compared to your new ones.
+
+Well in such cases BEM naming architecture for  styling comes to your rescue for providing an easy approach to name elements with lowest specificity (just by using classes only) and help write a CSS that is not only scalable but also maintanable as well.
+
+**Which companies use BEM naming Architecture?**  
+Web Development and Design agency like [SparkBox](https://seesparkbox.com/) use BEM during their development process. Read more about it [here](https://seesparkbox.com/foundry/bem_by_example).
+
+### Objectives
+
+**Participants will be able to:**
+- Understand what BEM naming architecture is and why it is used.
+- Understand how it helps with writing a scalable and maintanable CSS file.
+- Use BEM naming architecture on a small example project.
+
+### Materials
+
+The best reference/resource to get you started: 
+- [BEM documentaion](http://getbem.com/introduction/) (You can understand the basics of BEM -2 min read). 
+  
+
+### Lessons
+
+- To understand how BEM is used for styling elements, check out this great article on Mr-Frontend Blog: ['How to write better CSS with BEM'](https://mrfrontend.org/2017/10/write-better-css-with-bem/) by Raymon S (10 min read).
+
+
+### Common Mistakes / Misconceptions
+
+- **"My CSS styles never have conflicts since I utilize ID's most often to style my elements"**   
+Using ID's can definitely help you avoid any kind of specificity conflicts, since they have a priority over classes and tags, but that would mean, your styles are not reusable and you don't follow the DRY( Dont'Repeat Yourself) principle. New styles need to be written for every element even if some of them share a few common feature (Eg: buttons on a page, navlinks etc).
+
+### Guided Practice
+We'll make use of the BEM method to create few buttons and a hero-box .
+
+> Note: You must have prior knowledge of CSS and HTML to understand what BEM is and how to make use of it.
+
+**1.Understanding what BEM is?:** 
+
+BEM is a naming methodology/principle which stand for "Block", "Element", "Modifier" to describe and differentiate, the different components of your webpage in a way that is easy to understand,maintain and scale as and when required.
+In BEM methodology the important point to note is that each and every HTML element will be accessed using a class only, we don't use ID's or tags to style them, to keep specificity uniform across the entire project i.e. (0,1,0) : (ID's, Classes, Tags).
```

Comment:
I don’t quite understand the 010 part. If you’re just trying to express the true/false in a different form, can you use bullet points instead?

Reply:
I have made the changes so it becomes more understandable. Is it clear now?